docs/4.4/api-reference.md

```diff
@@ -11,57 +11,328 @@ Teleport is currently working on documenting our API.
 
         We are currently working on this project. If you have an API suggestion, [please complete our survey](https://docs.google.com/forms/d/1HPQu5Asg3lR0cu5crnLDhlvovGpFVIIbDMRvqclPhQg/edit).
 
+# gRPC APIs
+
+In most cases, you can interact with teleport using our cli tools, [tsh](cli-docs.md#tsh) and [tctl](cli-docs.md#tctl). However, there are some scenarios where you may need to interact with teleport programmatically. For this purpose, the same gRPC auth API that `tctl` and `tsh` use is available for direct usage.
```

Comment:
Teleport, CLI, and API should all be capitialized.

Also make it more generic explaining that the user will be using the same API that `tctl` and `tsh` use.